src/views/decorator.rs

```diff
@@ -13,6 +13,28 @@ use crate::{
 };
 
 pub trait Decorators: View + Sized {
+    /// Alter the style of the view.  
+    ///
+    /// -----
+    ///
+    /// Note: repeated applications of `style` overwrite previous styles.  
+    /// If you are returning from a function that produces a view, you may want
+    /// to use `base_style` for the returned [`View`] instead.  
+    /// ```rust
+    /// # use floem::{peniko::Color, view::View, views::{Decorators, label, stack}};
+    /// fn view() -> impl View {
+    ///     label(|| "Hello".to_string())
+    ///         .style(|| Style::default().font_size(20.0).color(Color::RED))
```

Comment:
should be `base_style` here?

Reply:
That's intended to show how `style` behaves if you use it on a view that's returned, to give an illustration of what goes wrong.
(Though I could see that as potentially being confusing, so perhaps it should just be the note?)

Reply:
Ah got you. 

Yeah maybe move "If you are returning from a function that produces a view, you may want to use `base_style` for the returned [`View`] instead. " under the code example.